bigtable/CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Bigtable client contribution guide
+
+For a complete guide to contributing to Google Cloud Go client libraries, view
+the [top-level contributing guide](../CONTRIBUTING.md).
+
+## Running Integration Tests
+The Bigtable integration tests will target the emulator by default. Some of the
+tests can only be run against production however. In order to do this you will
+need to specify a few command line flags.
```

Comment:
Does our continuous build run against prod or against the emulator? Might be good to add some indication about what the CI is doing.

Reply:
emulator right now. I am working to fix this though. Today our emulator tests run on presubmit as well which may/may not be the correct thing (they are pretty fast against emulator)

Reply:
https://github.com/googleapis/google-cloud-go/issues/4158